### PR TITLE
CC-14636 UnzerRestApi module enabler

### PR DESCRIPTION
**Pull request by kkicheglovspryker**
Ticket: https://spryker.atlassian.net/browse/CC-14636

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# UnzerRestApi Module
+[![CI](https://github.com/spryker-eco/unzer-rest-api/actions/workflows/ci.yml/badge.svg)](https://github.com/spryker-eco/unzer-rest-api/actions/workflows/ci.yml)
+[![Latest Stable Version](https://poser.pugx.org/spryker-eco/unzer-rest-api/v/stable.svg)](https://packagist.org/packages/spryker-eco/unzer-rest-api)
+[![License](https://img.shields.io/github/license/spryker-eco/unzer-rest-api.svg?b=master)](https://github.com/spryker-eco/unzer-rest-api)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-rest-api/build-status/master)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg)](https://php.net/)
+## Installation
```
Comment (AsonUnique) [inline README.md:11]: Please, add small module description

File: phpstan.neon [hunk 2]
```diff
@@ -0,0 +1,17 @@
+parameters:
+    level: 8
+    checkMissingIterableValueType: false
+    excludes_analyse:
+
+    bootstrapFiles:
```
Comment (AsonUnique) [inline phpstan.neon:6]: Please, check if you need this

File: tooling.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+architecture-sniffer:
+  priority: 2
```
Comment (AsonUnique) [inline tooling.yml:2]: Please, add code-sniffer too

File: README.md (hunk 1 above)
Comment (kkicheglovspryker) [reply README.md:11]: done

File: phpstan.neon (hunk 2 above)
Comment (kkicheglovspryker) [reply phpstan.neon:6]: yes. for github actions tests in future

File: tooling.yml (hunk 3 above)
Comment (kkicheglovspryker) [reply tooling.yml:2]: done

**Pull request merged**